app/assets/stylesheets/application.scss

```diff
@@ -92,4 +92,17 @@ $grey10: #C6CACC;
     padding-bottom: 5px;
     padding-left: 11px;
     padding-right: 11px;
+}
+
+.itemcard{
+    border: none;
+}
+
+.itemcard .card-body img{
+    border-radius: 5px;
+}
+
+.itemcard .card-body .card-title{
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
I think this shoud be in a different file. Maybe a itemcard.scss and then just put @import 'itemcard.scss' in this file.
For Comparison take a look at #132 application.scss

Reply:
That sounds good, the css now lives in `itemcard.css`.